### PR TITLE
Fix core bundling for IE 11

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Description
Closes part of https://github.com/department-of-veterans-affairs/va.gov-team/issues/34425

## Acceptance criteria
- [ ] Resolve IE11 issues with the core build

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/core/src/main.js [hunk 1]
```diff
@@ -2,4 +2,7 @@ import '@department-of-veterans-affairs/web-components/dist/component-library/co
 
 export * from '@department-of-veterans-affairs/react-components';
 export * from '@department-of-veterans-affairs/web-components';
-export { applyPolyfills } from '@department-of-veterans-affairs/web-components/loader';
+export {
+  applyPolyfills,
+  defineCustomElements,
```
Comment (bkjohnson) [inline packages/core/src/main.js:7]: Can you elaborate on this change? In https://github.com/department-of-veterans-affairs/component-library/pull/218 we switched from implicitly exporting everything (including `defineCustomElements`) to only explicitly exporting `applyPolyfills` in order to fix a webpack 5 issue in `vets-website`.

image

If the named export doesn't cause the same issue that the star export did then this is fine.

File: packages/core/webpack.config.js [hunk 2]
```diff
@@ -13,7 +13,6 @@ module.exports = {
     rules: [
       {
         test: /\.js$/,
-        exclude: /node_modules/,
```
Comment (bkjohnson) [inline packages/core/webpack.config.js:16]: Does this mean that we're now using babel to rebuild all of our npm dependencies?

**Review by bkjohnson — COMMENTED** [review]
I was able to make some changes and publish prereleases and eventually get things to work like we want on IE11: https://github.com/department-of-veterans-affairs/vets-website/pull/19908

With this PR I think we'll need to reset the versions back like we're ready to publish a real version. Once we double check everything and feel comfortable we should be good to go.

**Review by bkjohnson — APPROVED** [review]
We confirmed that IE11 worked with the `vets-website` changes using the changes in the prerelease.

**Comment by andresriveratoro** [comment]
Thank you for your work on this, Brooks.

**Pull request merged**